pgxn/neon/walproposer_pg.c

```diff
@@ -236,44 +250,44 @@ nwp_register_gucs(void)
 
 
 static int
-split_safekeepers_list(char *safekeepers_list, char *safekeepers[])
+split_safekeeper_connstrings(char *connstrings, char *safekeepers[])
 {
 	int			n_safekeepers = 0;
-	char	   *curr_sk = safekeepers_list;
+	char	   *curr_sk = connstrings;
 
-	for (char *coma = safekeepers_list; coma != NULL && *coma != '\0'; curr_sk = coma)
+	for (char *comma = connstrings; comma != NULL && *comma != '\0'; curr_sk = comma)
 	{
 		if (++n_safekeepers >= MAX_SAFEKEEPERS)
 		{
 			wpg_log(FATAL, "too many safekeepers");
 		}
 
-		coma = strchr(coma, ',');
+		comma = strchr(comma, ',');
 		safekeepers[n_safekeepers - 1] = curr_sk;
 
-		if (coma != NULL)
+		if (comma != NULL)
 		{
-			*coma++ = '\0';
+			*comma++ = '\0';
 		}
 	}
 
 	return n_safekeepers;
 }
 
 /*
- * Accept two coma-separated strings with list of safekeeper host:port addresses.
+ * Accept two comma-separated strings with list of safekeeper host:port addresses.
  * Split them into arrays and return false if two sets do not match, ignoring the order.
  */
 static bool
-safekeepers_cmp(char *old, char *new)
+safekeeper_connstrings_cmp(char *old, char *new)
```

Comment:
It does not take into account that `neon.safekeeper_connstrings` may start with generation prefix (but may be fixed in a separate PR, it's not related to current change)

Reply:
I'm not fixing that issue as a part of this PR.